README.md

```diff
@@ -106,6 +106,27 @@ The translated tokens annotations provides a itemList of translations for each o
 contains a domain identifying the language and script of the translations and a itemList of translations corresponding to
 the tokens of the text.
 
+# Serialization #
+
+The adm-json artifact provides support for reading and writing Json via
+the Jackson library. Custom serialization and deserialization provides
+type-safe lists and accomplishes polymorphism with a minumum of
+additional space.
+
+The artifact provides two Jackson modules. One writes the ADM as a
+graph of Json objects. The second avoids all the key names by using
+the 'array shape' feature of Jackson. In most cases, you can get
+space/time performance from the normal object model by using the
+Jackson 'SMILE' format or some other compression mechanism, but the
+array variation reduces overhead.
+
+The library has a concept of the version of the serialized form. When
+serializing, it adds a `version` property, and when deserializing it
+checks this version for compatibility. As of now, the format is at
+version 1.0.0. The deserialization code accepts either '1.x.y' or
```

Comment:
1.0.0 or 1.1.0?  We need to decide and then rename some things like
the "v0" in `json/test-data/simple_doc0_no_chains_adm_v0.json`
and the name of `CompatibilityV0ToV1`.